src/ontology/mondo-edit.obo

```diff
@@ -293933,20 +293913,36 @@ property_value: confidence "0.7021739130434781" xsd:double
 
 [Term]
 id: MONDO:0014700
-name: neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation
+name: Au-Klein syndrome
 subset: ordo_etiological_subtype {source="Orphanet:453504"}
-synonym: "AU-Kline syndrome" RELATED [MONDO:Lexical, OMIM:616580]
-synonym: "Au-Kline syndrome" EXACT [Orphanet:453504]
-synonym: "AUKS" RELATED ABBREVIATION [MONDO:Lexical, OMIM:616580]
-synonym: "neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation" EXACT CLINGEN_PREFERRED []
+subset: ordo_malformation_syndrome {source="Orphanet:2729"}
+synonym: "Au-Kline syndrome" EXACT [OMIM:616580, Orphanet:453504]
+synonym: "AUKS" EXACT ABBREVIATION [MONDO:Lexical, OMIM:616580]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and intellectual disability" RELATED [GARD:0004064]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and mental retardation" RELATED DEPRECATED [GARD:0004064]
```

Comment:
Why is this marked as `RELATED DEPRECATED`? I ask because it is shown here: https://rarediseases.org/gard-rare-disease/okamoto-syndrome/

Also, why are there 2 synonym axioms with `[GARD:0004064]` for both by one is `RELATED` (two lines above) and this one is `RELATED DEPRECATED`?

Reply:
here is some info about deprecated synonyms: https://mondo.readthedocs.io/en/latest/editors-guide/changelog-metadata-schema/#glossary

and here:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types

Reply:
Thanks, I see the reason now and also realize that `RELATED DEPRECATED` is actually two tags.